arcgis-ios-sdk-samples/Maps/Download preplanned map/DownloadPreplannedMap.storyboard

```diff
@@ -0,0 +1,157 @@
+                            <constraint firstAttribute="trailing" secondItem="pfV-1x-ZGz" secondAttribute="trailing" id="UOu-yd-DVv"/>
+                            <constraint firstItem="pfV-1x-ZGz" firstAttribute="height" secondItem="nE8-T5-4kh" secondAttribute="height" id="WrG-ZC-BiK"/>
+                            <constraint firstAttribute="trailing" secondItem="txe-YL-hFN" secondAttribute="trailing" id="YfO-RW-86T"/>
+                            <constraint firstItem="pfV-1x-ZGz" firstAttribute="leading" secondItem="nE8-T5-4kh" secondAttribute="leading" id="nKZ-Ye-Jkg"/>
+                            <constraint firstItem="txe-YL-hFN" firstAttribute="leading" secondItem="nE8-T5-4kh" secondAttribute="leading" id="pzh-up-HIi"/>
+                        </constraints>
+                        <viewLayoutGuide key="safeArea" id="YcI-mI-KS9"/>
+                    </view>
+                    <connections>
+                        <outlet property="activityIndicatorView" destination="9e3-Rl-Roi" id="sij-xr-JNO"/>
+                        <outlet property="mapView" destination="pfV-1x-ZGz" id="2FY-Mn-I2I"/>
+                        <outlet property="removeDownloadsBarButtonItem" destination="acg-XN-HC2" id="cj8-MC-ybC"/>
+                        <outlet property="selectMapBarButtonItem" destination="T6C-dm-3bb" id="xJb-4I-ml6"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="ZTH-Oj-W7H" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="39" y="21"/>
+        </scene>
+        <!--Select Map-->
+        <scene sceneID="ZnK-Kg-0rN">
+            <objects>
+                <tableViewController id="bfd-3L-KJe" customClass="MapSelectionTableViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="grouped" separatorStyle="none" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="18" sectionFooterHeight="18" id="h1N-TW-eBm">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <color key="backgroundColor" cocoaTouchSystemColor="groupTableViewBackgroundColor"/>
+                        <prototypes>
+                            <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" insetsLayoutMarginsFromSafeArea="NO" selectionStyle="none" hidesAccessoryWhenEditing="NO" indentationLevel="1" indentationWidth="0.0" reuseIdentifier="OnlineMapCell" textLabel="qgc-D9-kGY" style="IBUITableViewCellStyleDefault" id="mkd-v0-GDf">
```

Comment:
Did you intentionally disable selection? It seems odd not to see the cell highlight when touched. Usually selection is allowed so that the highlight is shown, but then deselect immediately upon selection.

Reply:
I did, as I thought "checkmark management" warranted it. I reconsidered this, and restored it in [cfdbce55](https://github.com/Esri/arcgis-runtime-samples-ios/pull/758/commits/cfdbce55e59d9bbfd52600a3c6ba3982cba698ca). I made a few small supporting changes as well.